### DISCUSSION TITLE
Add LocalAI

### DISCUSSION THREAD

**Discussion by aasmundkamphaug** [langflow-ai/langflow#681] [Ideas]
Hi it would be very nice if you could add LocalAi as an intergration, so it would be much easier to use self hosted Models

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
